src/main/java/com/transportcompany/transportcompany/services/UserServiceImpl.java

```diff
@@ -15,29 +17,29 @@
 @Service
 public class UserServiceImpl implements UserService {
 
-
-//List <userDTO> userDTOList =new ArrayList<>();
     private final UserRepository userRepository;
     private final ObjectMapper objectMapper;
 
     public UserServiceImpl(UserRepository userRepository, ObjectMapper objectMapper) {
 
         this.userRepository = userRepository;
         this.objectMapper = objectMapper;
-
     }
 
     @Override
     public UserDTO createUser(UserDTO userDTO) {
+        if (!userDTO.getEmail().contains("@")) {
```

Comment:
As an improvement, we can have this validation on a distinct method, maybe called `validateUserEmail`?